core/templates/dev/head/pages/Base.js

```diff
@@ -18,9 +18,13 @@
 
 oppia.controller('Base', [
   '$scope', '$rootScope', '$document', 'alertsService', 'SidebarStatusService',
-  function($scope, $rootScope, $document, alertsService, SidebarStatusService) {
+  'utilsService',
+  function(
+      $scope, $rootScope, $document, alertsService, SidebarStatusService,
+      utilsService) {
     $scope.alertsService = alertsService;
     $scope.currentLang = 'en';
+    $scope.utilsService = utilsService;
```

Comment:
I don't really understand why $scope.skipTo has to be in utilsService. Can't it just be defined within this controller instead, since this is the only place it's used?